### PR TITLE
Feat: Analytics integration

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

- integration of Segment analytics (with CI workflow adjustment)
- architecture of the isolated Analytics module, that needed to refactor bootstrapping logic
- implementation of most important analytics events
- in-editor settings to enable/disable analytic event or group of events

## How to test the changes?

Just check that it doesn't brake "happy path"

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/UserInAppInitializationFlow/IReadOnlyRealFlowLoadingStatus.cs [hunk 1]
```diff
@@ -1,7 +1,10 @@
+using DCL.Utilities;
+
 namespace DCL.UserInAppInitializationFlow
 {
     public interface IReadOnlyRealFlowLoadingStatus
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/UserInAppInitializationFlow/IReadOnlyRealFlowLoadingStatus.cs:5]: It's not read-only 😄

File: Explorer/Assets/DCL/Utilities/ReactiveProperties/ReadOnlyReactiveProperty.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿namespace DCL.Utilities
+{
+    public class ReadOnlyReactiveProperty<T> : IReadOnlyReactiveProperty<T>
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/Utilities/ReactiveProperties/ReadOnlyReactiveProperty.cs:3]: I think it is something strange that we will never need

File: Explorer/Assets/Scripts/Global/Static/StaticSceneLauncher.cs [hunk 3]
```diff
@@ -127,13 +127,16 @@ public async UniTask InitializationFlowAsync(CancellationToken ct)
...
             // First load the common global plugin
-            (StaticContainer staticContainer, bool isLoaded) = await StaticContainer.CreateAsync(
-                new NullDebugContainerBuilder(),
-                globalSettingsContainer,
-                web3IdentityProvider,
-                ethereumApi,
-                ct
-            )!;
+            (StaticContainer staticContainer, bool isLoaded) = (null, false);
+            //
```
Comment (mikhail-dcl) [inline Explorer/Assets/Scripts/Global/Static/StaticSceneLauncher.cs:131]: Please fix it

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]

**Review by anicalbano — APPROVED** [review]
🟢 Reviewed by QA

**Pull request merged**